ribbon/src/main/java/feign/ribbon/RibbonClient.java

```diff
@@ -59,4 +61,25 @@ private LBClient lbClient(String clientName) {
     ILoadBalancer lb = ClientFactory.getNamedLoadBalancer(clientName);
     return new LBClient(delegate, lb, config);
   }
+  
+  protected static class FeignOptionsClientConfig extends DefaultClientConfigImpl {
```

Comment:
nit: remove protected (as we don't expect this to be extended outside its package). also reformat to google java style using one of the below or similar

http://google-styleguide.googlecode.com/svn/trunk/eclipse-java-google-style.xml
http://google-styleguide.googlecode.com/svn/trunk/intellij-java-google-style.xml